tests/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ find_package(doctest REQUIRED)
 
 include("${doctest_DIR}/doctest.cmake")
 
-add_subdirectory("c_api_tests")
+add_subdirectory("api_tests")
```

Comment:
This is ambiguous because itbdoesbnotbtest the python API

Reply:
`native_api_tests`?

Reply:
yes, that's much better indeed

Reply:
It's now in the latest commit